test/IndexTestHelpers.h

```diff
@@ -59,6 +89,12 @@ inline Index makeTestIndex() {
 static inline QueryExecutionContext* getQec() {
   static ad_utility::AllocatorWithLimit<Id> alloc{
       ad_utility::makeAllocationMemoryLeftThreadsafeObject(100'000)};
+  static const absl::Cleanup cleanup = []() {
+    for (const std::string& indexFilename :
+         getAllIndexFilenames("_staticGlobalTestIndex")) {
```

Comment:
There are now two completely independent occurences of the magic string `"_staticGlobalTestIndex"` which introduces technical dept. 
My suggestion is to make the index basename an argument to the `makeTestIndex()`  function, then it is only used here.